DatadogInternal/Sources/Models/SessionReplay/PrivacyLevel+SessionReplay.swift

```diff
@@ -0,0 +1,81 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+#if os(iOS)
+import UIKit
+
+// MARK: - DatadogExtension for UIView
+
+/// Extension on `DatadogExtension` to provide access to `SessionReplayOverride` for any `UIView`.
+extension DatadogExtension where ExtendedType: UIView {
+    /// Provides access to Session Replay override settings for the view.
+    /// Usage: `myView.dd.sessionReplayOverride.textAndInputPrivacy = .maskNone`.
+    public var sessionReplayOverride: SessionReplayOverrideExtension<ExtendedType> {
+        get {
+            return SessionReplayOverrideExtension(self.type)
+        }
+        set {}
```

Comment:
IMO worth writing tests accessing feature through this API directly. In the end this the public API used by the customer

Reply:
In addition to the tests I created in SessionReplayOverrideTests? What more would you test?

Reply:
Missed it! thanks